yt_dlp/extractor/nfl.py

```diff
@@ -53,8 +53,7 @@ class NFLBaseIE(InfoExtractor):
                             )
                         )/
                     '''
-    _VIDEO_CONFIG_REGEX = r'<script[^>]+id="[^"]*video-config-[0-9a-f]{8}-(?:[0-9a-f]{4}-){3}[0-9a-f]{12}[^"]*"[^>]*>\s*({.+})'
```

Comment:
Would be better to use `self._search_json`, but unless u want do a refactor, this is good enough